src/components/Pages/Guide/GuideSectionCollection.js

```diff
@@ -101,13 +101,18 @@ function GuideSectionCollection({
           isMobileOrTablet={isMobileOrTablet}
           resizeCount={resizeCount}
         >
-          <GuideSection
-            page={page}
-            pageNumber={index + 1}
-            numberOfPages={section.pages.length}
-            sectionHeading={section.heading}
-            intl={intl}
-          />
+        { (page.servicePage || page.informationPage) &&
```

Comment:
Here, we'll break adding the content for the page if it dosen't have either a servicePage or an information page. This will keep pageless guide sections from breaking when built.